### DISCUSSION TITLE
[Metaphor] SCP Labels and Forms Standarization

### DISCUSSION THREAD

**Discussion by darkterminal** [StreetCommunityProgrammer/metaphore#26] [Ideas]
To simplify the distribution and management of content on the Metaphor Website, we need standardized labels to make it easier to process data from issues and pull requests that are made by members.

**Content Indicators**

- **metaphor** [`#4A9A4D`]: This label is used to indicate issues and pull requests related to the content on the Metaphor Website.
- **need to review** [`#C9636B`]: This label is used to indicate issues and pull requests that need to be reviewed by the relevant team members before they can be merged or published.
- **under review** [`#A055A4`]: This label is used to indicate issues and pull requests that are currently under review by the relevant team members.
- **published** [`#72D8C8`]: This label is used to indicate issues and pull requests that have been published on the Metaphor Website.

**Storyteller Indicators**
- **new storyteller** [`#F6A34E`]: This label is used to indicate issues and pull requests related to new storytellers who have expressed interest in contributing to the Metaphor Website.
- **storyteller verification** [`#EB1ED6`]: This label is used to indicate issues and pull requests related to the verification process of new storytellers.
- **verified** [`#5ABF7F`]: This label is used to indicate issues and pull requests related to storytellers who have been verified and approved to contribute to the Metaphor Website.

**Common labels used in Open Source repository issues are:**

- **bug**: Indicates that there is a problem with the software that needs to be fixed.
- **enhancement**: Indicates that there is a feature request or improvement that can be made to the software.
- **documentation**: Indicates that there is a need for documentation to be created or updated.
- **help wanted**: Indicates that the project maintainers are looking for help with the issue.
- **good first issue**: Indicates that the issue is a good entry point for new contributors to the project.
- **invalid**: Indicates that the issue is not valid or relevant to the project.
- **wontfix**: Indicates that the issue will not be fixed or implemented.
- **question**: Indicates that the issue is a question that needs to be answered.
- **duplicate**: Indicates that the issue is a duplicate of another issue and can be closed.
- **feature**: Indicates that the issue is a feature request that could be implemented in the future.

**Comment by mkubdev** [comment]
That's a great list! Everything looks good to me.
I don't have any objections about the colors used:

This is to better see the list of hexadecimal codes chosen:
![Metaphore labels colors](https://user-images.githubusercontent.com/50140834/222702755-671abef8-3a85-43b6-947f-8379e4f59a30.png)

**Reply by mkubdev** [reply]
Maybe `#A055A4` are too close from `#8C4CA7` ..

**Reply by darkterminal** [reply]
More better to visualized! 

Move `#BC4CA7` to `#EB1ED6`

**Reply by mkubdev** [reply]
![Frame 1](https://user-images.githubusercontent.com/50140834/222729731-cde4f5be-bb7f-40f4-bea0-786e8aac6cde.png)

@darkterminal much better!
